app/static/src/app/components/run/RunStochasticPlot.vue

```diff
@@ -29,6 +33,10 @@ export default defineComponent({
             type: Number,
             default: 0
```

Comment:
It makes a sort of sense because RunPlot and RunStochasticPlot will always have fitPlot=false, so should have a valid index. But I think it would make even more sense to make this a required prop!